podman-desktop.spec.rpkg

```diff
@@ -0,0 +1,91 @@
+# For automatic rebuilds in COPR
+
```

Comment:
I don't think it should belong in this repository.

Why do we need this rpkg file over the rpm that can be produced by electron-builder ?

This file probably don't need to be there
For example flathub (that is generating flatpak files contains all the information it its own repository https://github.com/flathub/io.podman_desktop.PodmanDesktop

Reply:
1. I have a separate [spec file](https://raw.githubusercontent.com/lsm5/podman-desktop-rpm/main/podman-desktop.spec) at [podman-desktop-rpm](https://github.com/lsm5/podman-desktop-rpm) which will generate a new RPM from every upstream release. Those rpms will get posted to a separate copr: [rhcontainerbot/podman-desktop](https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-desktop/builds/). That spec file is still far from being the official Fedora spec file but is a step in that direction. The rpm generated by electron-builder in my observation used FPM which can't be used for any official Fedora or RHEL inclusion. Nor would flatpak packages help towards that. So, we need rpm spec files if we're looking at any sort of official downstream inclusion.

2. This rpkg file serves a different purpose than the spec file mentioned in [1]. This rpkg spec file will build RPMs from the latest upstream main branch so that users looking for the latest bits can try them out quickly and
conveniently. This is already being done upstream for [Podman](https://github.com/containers/podman/blob/main/podman.spec.rpkg), [Buildah](https://github.com/containers/buildah/blob/main/buildah.spec.rpkg), [Skopeo](https://github.com/containers/skopeo/blob/main/skopeo.spec.rpkg), [Netavark](https://github.com/containers/netavark/blob/main/netavark.spec.rpkg), and [Aardvark-dns](https://github.com/containers/aardvark-dns/blob/main/aardvark-dns.spec.rpkg). All those RPMs are available at a [single COPR repo](https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-next/packages/), so it's a 1-stop shop for people looking to use the very latest container tools. There's at least 1 RH customer using the `rhcontainerbot/podman-next` COPR for early product integration. Also, see a relevant [blog post](https://hobo.house/2017/09/03/automate-rpm-builds-from-git-sources-using-copr/).

3. Electron-builder will need every user to build the rpm themselves every time they need to update to the latest. This rpkg file takes that hassle away. Users will only need to enable 1 single COPR and then just  `dnf update` to get the latest of everything.

4. This rpkg file comes with effectively 0 additional cost to upstream. It does not affect the upstream PR workflow, and works on the fedora COPR infra only after the PR is merged. Also, with this PR, I'm effectively signing up to maintain that file and the COPR builds myself. So, it shouldn't be any extra burden on the current
upstream maintainers.

Reply:
so you convinced me that it should not be part of this repository as it's not buildable outside of a dedicated environment.

Reply:
> so you convinced me that it should not be part of this repository as it's not buildable outside of a dedicated environment.

well, it's buildable using `rpkg local` on anyone's fedora installation or any distro that ships rpkg for that matter. So, a dedicated environment is not needed for that.